app/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+const errors = require('../errors'),
+  logger = require('../logger'),
+  jwt = require('jwt-simple'),
+  secret = process.env.NODE_API_JWT_SECRET_STRING,
+  User = require('../models').users;
+
+exports.checkCredentials = (req, res, next) => {
+
+  if (!req.headers.token){
+    return res.status(401).send('You must be logged in to access this endpoint');
+  }
+
+  let token = jwt.decode(req.headers.token, secret);
+
+  User.findOne({
+    where: {
+      email: token.token
```

Comment:
I just realized that you are using the attribute token to define the user email. Here you can see how that doesn't seeme right, dont you?

When you are encoding the token at sign_in time you should do something like:
`const token = jwt.encode({email: result.email}, config.common.session.secret);`